tests/unittests/test_util.py

```diff
@@ -2171,7 +2171,7 @@ def test_none_returns_default(self):
         )
 
 
-class TestMountinfoParsing(helpers.ResourceUsingTestCase):
+class TestMountinfoParsing(helpers.CiTestCase):
```

Comment:
Same here, is it worth followup as a separate PR to from CiTestCase here too?

Reply:
Done.